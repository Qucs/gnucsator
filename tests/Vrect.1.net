# Qucs 0.0.19  /home/felix/extern/gnucap-plugins/tests/simple.sch

R:R2 _net0 gnd R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R1 _net1 _net0 R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
VProbe:Pr1 _net0 gnd
VProbe:Pr0 _net1 gnd
Vrect:V1 _net1 gnd U="1 V" TH="1 ms" TL="1 ms" Tr="1 ns" Tf="1 ns" Td="0 ns"

.TR:TR1 Type="lin" Start="0" Stop="10m" Points="11" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
