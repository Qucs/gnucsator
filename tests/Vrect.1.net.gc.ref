<Qucs Dataset 0.0.19>
<indep time 21>
  +0.
  +1.00000000000000022n
  +0.00100000299999999953
  +0.00100000399999999967
  +0.00200000200000000027
  +0.00200000299999999997
  +0.00300000499999999936
  +0.0030000059999999995
  +0.00400000400000000055
  +0.00400000500000000069
  +0.00500000700000000009
  +0.00500000799999999934
  +0.00600000599999999995
  +0.00600000700000000009
  +0.00700000900000000036
  +0.00700000999999999962
  +0.00800000800000000111
  +0.00800000899999999859
  +0.00900001099999999887
  +0.0090000119999999999
  +0.0100000000000000022
</indep>
<dep _net0.Vt time>
  +0.
  +0.120563520549119007
  +0.120563520549119007
  +52.8490000000000037p
  +0.
  -0.130875026003500005
  +0.120563520549119007
  +52.8490000000000037p
  +0.
  +1.04575392264558897
  +0.120563520549119007
  +52.8490000000000037p
  +0.
  +1.04575392264558897
  +0.120563520549119007
  +52.8490000000000037p
  +0.
  -1.3075044453041682
  +0.120563520549119007
  -1.50700800000000034n
  +0.
</dep>
<dep _net1.Vt time>
  +0.
  +0.620563470552137985
  +0.620563470552137985
  +99.1870000000000118p
  +0.
  +0.369124923946896066
  +0.620563470552137985
  +99.1870000000000118p
  +0.
  +1.54575387284224134
  +0.620563470552137985
  +99.1870000000000118p
  +0.
  +1.54575387284224134
  +0.620563470552137985
  +99.1870000000000118p
  +0.
  -0.807504495600027993
  +0.620563470552137985
  -1.02699000000000006n
  +0.
</dep>
<dep gnd.Vt time>
  +0.
  -0.379436429447872036
  -0.379436429447872036
  +6.51200000000000134p
  +0.
  -0.630874975960439066
  -0.379436429447872036
  +6.51200000000000134p
  +0.
  +0.545753972501224993
  -0.379436429447872036
  +6.51200000000000134p
  +0.
  +0.545753972501224993
  -0.379436429447872036
  +6.51200000000000134p
  +0.
  -1.80750439507368216
  -0.379436429447872036
  -1.98702700000000014n
  +0.
</dep>
<dep Pr1.Vt time>
  +0.
  +0.499999949996990977
  +0.499999949996990977
  +46.337513507463699p
  -26.769694883981505E-18
  +0.49999994995693946
  +0.499999949996990977
  +46.337513507463699p
  -26.769694883981505E-18
  +0.499999950144364024
  +0.499999949996990977
  +46.337513507463699p
  -26.769694883981505E-18
  +0.499999950144364024
  +0.499999949996990977
  +46.337513507463699p
  -26.769694883981505E-18
  +0.499999949769514451
  +0.499999949996990977
  +480.01837718661191p
  +11.244337735660228E-18
</dep>
<dep Pr0.Vt time>
  +0.
  +0.999999900000009844
  +0.999999900000009844
  +92.6750270175698886p
  -53.539389768643905E-18
  +0.999999899907334999
  +0.999999900000009844
  +92.6750270175698886p
  -53.539389768643905E-18
  +0.999999900341015646
  +0.999999900000009844
  +92.6750270175698886p
  -53.539389768643905E-18
  +0.999999900341015646
  +0.999999900000009844
  +92.6750270175698886p
  -53.539389768643905E-18
  +0.999999899473653997
  +0.999999900000009844
  +960.036754297873429p
  +22.488675473464896E-18
</dep>
