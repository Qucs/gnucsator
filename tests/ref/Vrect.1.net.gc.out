<Qucs Dataset>
default plugins: ignored
Gnucap   System status
iterations: op=0, dc=0, tran=232, fourier=0, total=325
transient timesteps: accepted=61, rejected=32, total=93
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=2, total=4
dctran density=50.0%, ac density=50.0%

<indep time 61>
 0.
 1.01E-9
 2.02E-9
 3.03E-9
 4.04E-9
 5.05E-9
 0.001000001
 0.001000002
 0.001000004
 0.001000004
 0.001000004
 0.001000004
 0.002000002
 0.002000003
 0.002005434
 0.002006745
 0.002007105
 0.002007464
 0.003000003
 0.003000004
 0.003000006
 0.003000006
 0.003000006
 0.003000006
 0.004000004
 0.004000005
 0.004005436
 0.004006747
 0.004007107
 0.004007466
 0.005000005
 0.005000006
 0.005000008
 0.005000008
 0.005000008
 0.005000008
 0.006000006
 0.006000007
 0.006005438
 0.006006749
 0.006007109
 0.006007468
 0.007000007
 0.007000008
 0.00700001
 0.00700001
 0.00700001
 0.00700001
 0.008000008
 0.008000009
 0.00800544
 0.008006751
 0.008007111
 0.00800747
 0.009000009
 0.00900001
 0.009000012
 0.009000012
 0.009000012
 0.009000012
 0.01
</indep>
<dep _net0.Vt time>
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
</dep>
<dep _net1.Vt time>
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
</dep>
<dep Pr1.Vt time>
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.4999999
 0.
 0.
 0.
 0.
 0.
 0.
</dep>
<dep Pr0.Vt time>
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.9999999
 0.
 0.
 0.
 0.
 0.
 0.
</dep>
